activerecord/lib/active_record/attribute_methods.rb

```diff
@@ -19,6 +19,11 @@ module AttributeMethods
       include TimeZoneConversion
       include Dirty
       include Serialization
+
+      # Provides access to the value of the record's id column.
+      # Use +uid+ over +id+ if the raw id column value is needed,
+      # rather than the value of the primary key.
+      alias_attribute :uid, :id
```

Comment:
I think this is implemented in wrong place. It should be called in the same place that we call `attribute :name` for example. (I believe it is in the schema loader)

Reply:
I wasn't entirely sure where you were suggesting this be moved 😅  I moved it to `#define_attribute`, since we define all of the attributes for a model when `load_schema!` is called. Made sense that we'd register the alias when defining the `id` attribute. Let me know what you think.